Add helper to create toolbar button from command

vidartf opened the pull request:
Creates a toolbar button from a command, and listens for changes.

### TODO
 - [x] Add unit tests
 - [x] ~~Use for existing toolbar buttons with a corresponding command~~ (cannot use commands internally, to support use as library)

blink1073 commented on packages/apputils/src/toolbar.ts:
```diff
@@ -195,6 +199,44 @@ class Toolbar<T extends Widget> extends Widget {
+        button.node.title = Private.commandTooltip(commands, id);
+      }
+    }
+    commands.commandChanged.connect(onChange);
```
If you change this to `connect(onChange, button)` then it will automatically be disconnected when the button is disposed.

vidartf replied:
Ooh, nice! It would be neat if this was stated in the docstring of `connect`.

blink1073 replied:
It is only true in this case because the base widget class dispose method clears its signal data.

blink1073 commented on packages/apputils/src/toolbar.ts:
```diff
@@ -195,6 +199,44 @@ class Toolbar<T extends Widget> extends Widget {
+   */
+  export
+  function toolbarButtonFromCommand(commands: CommandRegistry, id: string): ToolbarButton {
+    let oldClass = commands.className(id)
```
missing a semicolon

vidartf commented:
I fixed the previous feedback, and added some more features. I haven't been able to test it live though, because I'm struggling with making an environment in which to test it.

blink1073 commented:
What specifically is the problem?

blink1073 commented:
Looks good, merging so it gets in the 0.25 release.  Thanks!
